plugins/bulk-import/src/components/AddRepositories/AddRepositoriesPage.tsx

```diff
@@ -2,118 +2,103 @@ import React from 'react';
       <Header title="Add repositories" type="Bulk import" typeLink=".." />
-      <Content>
-        <Accordion defaultExpanded>
-          <AccordionSummary
-            expandIcon={<ExpandMoreIcon />}
-            id="add-repository-summary"
-          >
-            <Typography variant="h5">
-              Add repositories to Red Hat Developer Hub in 5 steps
-            </Typography>
-          </AccordionSummary>
-          <AccordionDetails
-            id="add-repository-illustrations"
-            className={styles.illustration}
-          >
-            <Illustrations
-              iconClassname={
-                theme.palette.type === 'dark'
-                  ? 'icon-approval-tool-white'
-                  : 'icon-approval-tool-black'
-              }
-              iconText="Choose approval tool (git/ServiceNow) for PR/ticket creation"
-            />
-            <Illustrations
-              iconClassname={
-                theme.palette.type === 'dark'
-                  ? 'icon-choose-repositories-white'
-                  : 'icon-choose-repositories-black'
-              }
-              iconText="Choose repositories you want to add"
-            />
-            <Illustrations
-              iconClassname={
-                theme.palette.type === 'dark'
-                  ? 'icon-generate-cataloginfo-white'
-                  : 'icon-generate-cataloginfo-black'
-              }
-              iconText="Generate a catalog-info.yaml file for each repository"
-            />
-            <Illustrations
-              iconClassname={
-                theme.palette.type === 'dark'
-                  ? 'icon-edit-pullrequest-white'
-                  : 'icon-edit-pullrequest-black'
-              }
-              iconText="Edit the pull request details if needed"
-            />
-            <Illustrations
-              iconClassname={
-                theme.palette.type === 'dark'
-                  ? 'icon-track-status-white'
-                  : 'icon-track-status-black'
-              }
-              iconText="Track the approval status"
-            />
-          </AccordionDetails>
-        </Accordion>
-        <br />
-        <AddRepositoriesForm
-          values={formik.values}
-          setFieldValue={formik.setFieldValue}
-          setapprovalTool={setApprovalTool}
-        />
+      <Content noPadding>
+        <div style={{ padding: '24px' }}>
+          <Accordion defaultExpanded>
+            <AccordionSummary
+              expandIcon={<ExpandMoreIcon />}
+              id="add-repository-summary"
+            >
+              <Typography variant="h5">
+                Add repositories to Red Hat Developer Hub in 5 steps
+              </Typography>
+            </AccordionSummary>
+            <AccordionDetails
+              sx={{
+                flexDirection: 'row',
+                display: 'flex',
+                justifyContent: 'space-around',
+                overflow: 'scroll',
```

Comment:
```suggestion
                overflow: 'auto',
```

i can see scroll, always

<img width="1624" alt="image" src="https://github.com/janus-idp/backstage-plugins/assets/5129024/d67bcc4f-e907-4411-a28e-4dd843c9cd2f">